docs/csharp/package-management.md

```diff
@@ -49,13 +59,13 @@ In C# Dev Kit, you can perform NuGet package operations in either of two ways: b
 
 2. If you have more than one project in your solution, you are asked to select which project contains the package that you would like to update.
 
-3. Next, choose which package to update from the list of packages that are currently installed in your project and that have an update available. If no packages in your solution have an update available, you see a message notifying you of this.
+3. Next, choose which package to update from the list of packages that are currently installed in your project and that have an update available. If no packages in your solution have an update available, you will see a message notifying you of this.
```

Comment:
```suggestion
3. Next, choose which package to update from the list of packages that are currently installed in your project and that have an update available. If no packages in your solution have an update available, you see a message notifying you of this.
```

Reply:
RE: you will see vs. you see

"You see" denotes it is appearing right now.

"You will see " is better grammar & makes more sense from a user standpoint

Reply:
In this context (product docs), this phrase refers to a _real condition_, where every time the condition is met, the outcome is always true. In technical docs we want to be as much "in the moment" as possible.